Packages/com.unity.inputsystem/InputSystem/Plugins/UI/TrackedDeviceRaycaster.cs

```diff
@@ -226,6 +240,39 @@ private bool RayIntersectsRectTransform(RectTransform transform, Ray ray, out Ve
             distance = 0;
             return false;
         }
+
+        [FormerlySerializedAs("ignoreReversedGraphics")]
+        [SerializeField]
+        private bool m_IgnoreReversedGraphics;
+
+        [FormerlySerializedAs("checkFor2DOcclusion")]
+        [SerializeField]
+        private bool m_CheckFor2DOcclusion;
+
+        [FormerlySerializedAs("checkFor3DOcclusion")]
+        [SerializeField]
+        private bool m_CheckFor3DOcclusion;
+
+        [Tooltip("Maximum distance (in 3D world space) that rays are traced to find a hit.")]
```

Comment:
Are we localizing Input System? We may want to localize these tooltips.

Reply:
Do we support localization for TooltipAttributes?

In general, while there's some EditorGUIUtility.TrTextContent calls are in the UI frontend, no serious effort has yet been put into localization. I'd like to see this tackled after 1.0. For player stuff (like rebind screens) as well as for in-editor UIs.

Reply:
We do have support for localizing packages. We can look into it after we leave preview.

Reply:
Let's do it :)